.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,146 @@
+name: "Prepare CodeQL Coding Standards release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: |
+          The version to release (MUST follow semantic versioning).
```

Comment:
Can you annotate whether these require `v` prefix or not for clarity.

Reply:
Done